client/whiteboard/sketchpad.tsx

```diff
@@ -99,6 +111,17 @@ export const Sketchpad: React.FC<Props> = ({
     >
       {children}
       <Layer>
+        {[...state.userPositions.values()].map(userPosition => (
+          <Text
+            text={userPosition.userName}
+            x = {userPosition.point.x}
+            y = {userPosition.point.y}
+            key={userPosition.id}
+            fontSize={15}
+            fontFamily={"Calibri"}
+            fill={"#34ebc0"}
```

Comment:
the active user is assigned a color - might be cool to use that here so it matches how they show up in chat, wdyt?

Reply:
i wasn't aware of that, sure I can use the active user color